acme/acme/client_test.py

```diff
@@ -621,6 +622,26 @@ def test_requests_error_passthrough(self, mock_requests):
         self.assertRaises(requests.exceptions.RequestException,
                           self.net._send_request, 'GET', 'uri')
 
+    def test_urllib_error(self):
+        try:
+            # pylint: disable=protected-access
+            self.net._send_request('GET', "http://localhost:19123/nonexistent.txt")
+
+        except ValueError as y:
+            if "linux" in sys.platform:
+                self.assertEqual("Requesting localhost/nonexistent: "
+                                 "[Errno 111] Connection refused", str(y))
```

Comment:
I thought about it a bit more and it might be helpful for the end user to see the error number - they may have seen that urllib3 error message before.  It's easy enough for me to remove if we decide that we should, maybe we should just decide that together.